src/Model/Command/UserRecommendation.php

```diff
@@ -134,6 +137,22 @@ public function setFilters(array $filters): self
         return $this;
     }
 
+    /**
+     * Specify the filter type being used.
+     */
+    public function setFilterType(string $filterType): self
```

Comment:
This whole method seems uncovered by unit tests. I think there should be (simple) testcase for the rgx type.

Reply:
After discussion, we've removed this one :)